src/Microsoft.Diagnostics.Monitoring.EventPipe/Triggers/EventCounter/EventCounterTriggerSettings.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace Microsoft.Diagnostics.Monitoring.EventPipe.Triggers.EventCounter
+{
+    /// <summary>
+    /// The settings for the <see cref="EventCounterTrigger"/>.
+    /// </summary>
+    internal sealed class EventCounterTriggerSettings :
+        IValidatableObject
+    {
+        internal const int CounterIntervalSeconds_MaxValue = 24 * 60 * 60; // 1 day
+        internal const int CounterIntervalSeconds_MinValue = 1; // 1 second
+
+        internal const string EitherGreaterThanLessThanMessage = "Either the " + nameof(GreaterThan) + " field or the " + nameof(LessThan) + " field are required.";
```

Comment:
>EitherGreaterThanLessThanMessage [](http://example.com/codeflow?start=30&length=32)

Will these strings need to be localized in the future?

Reply:
As how the repo is currently unlocalizable (e.g. there are no resource files for localization), I would say these will probably not be localized in the future.